extensions/notebook/src/jupyter/jupyterServerInstallation.ts

```diff
@@ -53,11 +58,43 @@ export interface IJupyterServerInstallation {
 	getCondaExePath(): string;
 	executeBufferedCommand(command: string): Promise<string>;
 	executeStreamedCommand(command: string): Promise<void>;
-	installPipPackages(packages: PythonPkgDetails[], useMinVersion: boolean): Promise<void>;
+	/**
+	 * Installs the specified packages using pip
+	 * @param packages The list of packages to install
+	 * @param useMinVersionDefault Whether we install each package as a min version (>=) or exact version (==) by default
+	 */
+	installPipPackages(packages: PythonPkgDetails[], useMinVersionDefault: boolean): Promise<void>;
 	uninstallPipPackages(packages: PythonPkgDetails[]): Promise<void>;
 	pythonExecutable: string;
 	pythonInstallationPath: string;
 }
+
+export const jupyterPkg: PythonPkgDetails = {
```

Comment:
Since these are now public constants, do we want to prepend "required" to the front of each so that it's easier to tell what they are from intellisense?

Reply:
Done.